Configuration/AlCa/python/autoCond.py

```diff
@@ -52,23 +52,23 @@
     # GlobalTag for MC production (cosmics) with realistic alignment and calibrations for Phase1 2017 detector, Strip tracker in PEAK mode
     'phase1_2017_cosmics_peak' :  '106X_mc2017cosmics_realistic_peak_v2',
     # GlobalTag for MC production with perfectly aligned and calibrated detector for full Phase1 2018 (and 0,0,0-centred beamspot)
-    'phase1_2018_design'       :  '106X_upgrade2018_design_v5',
+    'phase1_2018_design'       :  '106X_upgrade2018_design_v6',
     # GlobalTag for MC production with realistic conditions for full Phase1 2018 detector
-    'phase1_2018_realistic'    :  '106X_upgrade2018_realistic_v8',
+    'phase1_2018_realistic'    :  '106X_upgrade2018_realistic_v11',
     # GlobalTag for MC production with realistic conditions for full Phase1 2018 detector for Heavy Ion
-    'phase1_2018_realistic_hi' :  '106X_upgrade2018_realistic_HI_v5',
+    'phase1_2018_realistic_hi' :  '106X_upgrade2018_realistic_HI_v6',
     # GlobalTag for MC production with realistic conditions for full Phase1 2018 detector: HEM-15-16 fail
-    'phase1_2018_realistic_HEfail' :  '106X_upgrade2018_realistic_HEfail_v8',
+    'phase1_2018_realistic_HEfail' :  '106X_upgrade2018_realistic_HEfail_v11',
     # GlobalTag for MC production (cosmics) with realistic conditions for full Phase1 2018 detector,  Strip tracker in DECO mode
-    'phase1_2018_cosmics'      :   '106X_upgrade2018cosmics_realistic_deco_v4',
+    'phase1_2018_cosmics'      :   '106X_upgrade2018cosmics_realistic_deco_v7',
     # GlobalTag for MC production (cosmics) with realistic conditions for full Phase1 2018 detector,  Strip tracker in PEAK mode
-    'phase1_2018_cosmics_peak' :   '106X_upgrade2018cosmics_realistic_peak_v4',
+    'phase1_2018_cosmics_peak' :   '106X_upgrade2018cosmics_realistic_peak_v7',
```

Comment:
@tocheng @christopheralanwest 
```
$ conddb diff 106X_upgrade2018cosmics_realistic_peak_v7 106X_upgrade2018cosmics_realistic_peak_v4 | grep SiStrip
SiStripApvGain2Rcd    -                                                 SiStripApvGain_FromParticles_Realistic2018MC_v0_mc  SiStripApvGain_FromParticles_DecoMode_v6_mc           
SiStripApvGainRcd     -                                                 SiStripApvGain_Realistic2018MC_v0_mc                SiStripApvGain_RealisticMC_v2_mc                      
SiStripApvGainSimRcd  -                                                 SiStripApvGain_Realistic2018MCSim_v0_mc             SiStripApvGain_RealisticMCSim_v3_mc                   
SiStripNoisesRcd      -                                                 SiStripNoise_DecoMode_2018_Minus20C_v0_mc           SiStripNoise_PeakMode_TickCorr_Minus15C_Fixed_v2_mc   
```
as we've discussed via e-mail `SiStripNoise_DecoMode_2018_Minus20C_v0_mc` can't be used in the peak GT, please use `SiStripNoise_PeakMode_2018_Minus20C_v0_mc` instead.